netci.groovy

```diff
@@ -32,14 +32,20 @@ def branch = GithubBranchName
                 case 'Windows_NT':
                     newJob.with{
                         steps{
-                            def windowsScript = "call \"C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Enterprise\\Common7\\Tools\\VsDevCmd.bat\" && cibuild.cmd --target ${runtime}"
+                            // all windows builds do a full framework localized build to produce satellite assemblies
+                            def script = "call \"C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Enterprise\\Common7\\Tools\\VsDevCmd.bat\" && cibuild.cmd --target Full --localized-build"
 
-                            // only Desktop support localized builds 
-                            if (runtime == "Desktop") {
-                                windowsScript += " --localized-build"
+                            if (runtime == "Full") {
+                                script += " --scope Test"
+                            }
+                            // .net core builds are localized (they need the satellites from the full framework build), run tests, and also build the nuget packages
+                            else if (runtime == "CoreCLR") {
+                                script == " --scope Compile --build-only"
+                                script += " && cibuild.cmd --target CoreCLR --scope test --localized-Build"
+                                script += " && msbuild build/NugetPackages/CreateNuGetPackages.proj"
```

Comment:
The baked in version would be `cibuild.cmd --target All --scope Test --localized-build && msbuild build/NugetPackages/CreateNuGetPackages.proj`
The problem there is that `--target All --scope Test` runs tests for both frameworks. I could change the script to this and then on windows jobs do not disambiguate between Full and Core as there's a command to rule them all.

Alternative is to have `cibuild.cmd --WindowsCoreLocalizedJob` which rewrites to `cibuild.cmd --target Full --scope Compile --build-only --localized-build && cibuild.cmd --target CoreCLR --scope test --localized-Build && msbuild build/NugetPackages/CreateNuGetPackages.proj`

The first option is less code, but when something fails we don't know whether it's FF or Core. The second option adds more complexity to the script. Second option then? :)